proto/redpanda/api/console/v1alpha1/debug_bundle.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto3";
+
+package redpanda.api.console.v1alpha1;
+
+message CreateDebugBundleRequest {
+  int32 controller_logs_size_limit_bytes = 1;
+  int32 cpu_profiler_wait_seconds = 2;
+  string logs_since = 3; // TODO add validation 'YYYY-MM-DD', 'yesterday', or 'today'
+  int32 logs_size_limit_bytes = 4;
+  string logs_until = 5; // TODO add validation 'YYYY-MM-DD', 'yesterday', or 'today'
+  int32 metrics_interval_seconds = 6;
+  repeated string partition = 7;
+}
+
+message CreateDebugBundleResponse {
+  string job_id = 1; // UUID
+}
+
+message GetDebugBundleStatusRequest {}
+
+message GetDebugBundleStatusResponse {
+  string job_id = 1;
+  Status status = 2;
+
```

Comment:
missing:
* start time
* file name
* stdout (list of strings)
* stderr (list of strings)

Reply:
Hmm did you review an outdated version or commit somehow?

```proto
// The response to GetDebugBundleStatus.
message GetDebugBundleStatusResponse {
  repeated DebugBundleStatus statuses = 1;
}
```

Where 

```proto
message DebugBundleStatus {
  // The broker ID.
  int32 broker_id = 1;

  // The job UUID of this process.
  string job_id = 2;

  // The status of the job.
  Status status = 3;

  // When the job was started.
  google.protobuf.Timestamp created_at = 4;

  // Path in API to get the file.
  string filename = 5;

  // Only filled in once the process completes.  Content of stdout from rpk.
  repeated string stdout = 6;

  // Only filled in once the process completes.  Content of stderr from rpk.
  repeated string stderr = 7;

  enum Status {
    STATUS_UNSPECIFIED = 0;
    STATUS_RUNNING = 1;
    STATUS_SUCCESS = 2;
    STATUS_ERROR = 3;
  }
}
```